src/Branding.svelte

```diff
@@ -0,0 +1,5 @@
+<div class="branding whitespace-normal">
```

Comment:
I don't believe this has been styled yet 😂

Reply:
I removed the reference to this component in App.svelte.